aws/cloudformation/domain_redirect.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+# verbose=0
+# while getopts "v" opt
+# do
+#     case $opt in
+#     (v) verbose=1 ;;
+#     (*) printf "Illegal option '-%s'\n" "$opt" && exit 1 ;;
+#     esac
+# done
+
+# Get the options
+verbose=0
+while getopts ":vd:t:" option; do
+  case $option in
+    v) # display Help
```

Comment:
is this comment accurate? Looks to me like this displays the generated `.yml`, but doesn't provide instructions on how to use the script, which is what I'd expect from a "Help" option

Reply:
fixed!